user-service/src/main/java/by/battle/userservice/dto/UserDto.java

```diff
@@ -0,0 +1,31 @@
+package by.battle.userservice.dto;
+
+import by.battle.userservice.entity.Role;
+import by.battle.userservice.entity.Status;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDto {
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("password")
```

Comment:
you may not specify JsonProperty, jackson will do this for you. there might be exceptions, but you will see them if you faced with such .